### DISCUSSION TITLE
Blender files for Avatar B/C

### DISCUSSION THREAD

**Discussion by markfoodyburton** [mozilla/hubs#3055] [General Questions & Discussion] [answered]
Are the blender files available for Avatar B/C (which seem to be Mozilla's, as they have Mozilla T-shirts, though I guess that proves nothing, but they are 'featured'...)

**Comment by matthewbcool** [comment, marked as answer]
https://github.com/MozillaReality/hubs-avatar-pipelines/tree/master/Blender/AvatarBot

**Reply by markfoodyburton** [reply]
Thanks for the swift reply, I'm aware of the "robot" avatar files, I was asking if the 'Avatar B/C' had been made available as a starting point. It is quite significantly different from the robot, and would be a great starting point for other avatars.

**Reply by j-conrad** [reply]
We haven’t made them available yet as they’re being redone.

**Reply by joone-org** [reply]
Is it possible to access the GLB from the URL?  Specifically this one:  https://hubs.mozilla.com/avatars/5wOX9YN.  I'd like to put my logo on it rather than Mozilla's, if that's allowed?

**Reply by joone-org** [reply]
https://hubs.mozilla.com/avatars/5wOX9YN + something something (maybe src/assets/models/DefaultAvatar.glb) ?
